swan-lake/learn-the-platform/native-support/build-a-native-executable.md

```diff
@@ -60,12 +70,12 @@ To complete this part of the guide, you need:
 ### Configure GraalVM
 
 1. Install GraalVM if you have not done it already.
-    - Install GraalVM with one line
+    - Install GraalVM on Linux, macOS, and Windows(via Git Bash, Cygwin, or WSL) with one line
```

Comment:
```suggestion
    - Install GraalVM on Linux, macOS, and Windows (via Git Bash, Cygwin, or WSL) with one line
```
What do we mean by `one line` here? Better if we can explain it more.

Reply:
The command below is a single command to install GraalVM and native-image tool. I think we don't need to emphasize it here. We can just say : 
```suggestion
    - Execute the following command to install GraalVM on Linux, macOS, and Windows(via Git Bash, Cygwin, or WSL)
```
@shafreenAnfar @praneesha WDYT?

Reply:
Let's just mention the command. No need to say `one line`.

Reply:
@praneesha Fixed via [92ebef7](https://github.com/ballerina-platform/ballerina-dev-website/pull/5905/commits/92ebef771a0c2052d07c49cd872ac69bcccd230b)